src/test/ui/traits/vtable/vtable-hrtb.rs

```diff
@@ -0,0 +1,39 @@
+// build-fail
+#![feature(rustc_attrs)]
+
+// issue 90177
+
+#[rustc_dump_vtable]
```

Comment:
I don't think we need to have this here; this is more of a "comparing trait refs with 'identical' bounds" issue than a vtable one.

Probably can move the test to hrtbs directory too.

Reply:
Updated and moved the test.